functions/src/routes/experiences/experienceHistory.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable promise/no-nesting */
+const admin = require("firebase-admin");
+
+const db = admin.firestore();
+
+const experienceHistory = (req, res) => {
+    const {id: userId} = req.payload;
+    const {type = 'all'} = req.query;
+    let experienceRef = db.collection("experiences");
+
+    if (type === 'current') {
+        experienceRef = experienceRef
+            .where("allAttendees", "array-contains", userId)
```

Comment:
Use currentAttendee here, because we are trying to get the active experience of the user

Reply:
Alright